prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics.yaml

```diff
@@ -399,6 +399,9 @@ periodics:
       - 'GKE_MACHINE_TYPE=n2-standard-4'
       - 'GKE_DISK_SIZE=25Gb'
       - 'E2E_ARGS=--stress'
+      resources:
+        requests:
+          memory: "20Gi" # Stress tests watch many large objects
```

Comment:
Suggest setting a cpu request as well.

Given the shape of the nodes I'd suggest something like 6cpu / 24Gi mem

Reply:
done